src/components/Community/styles.scss

```diff
@@ -0,0 +1,49 @@
+.community {
+  &__socials {
+    display: flex;
+    justify-content: center;
+    gap: 24px;
+    width: 100%;
+
+    &-item {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      gap: 30px;
+      background-color: var(--secondary-control-color-40);
+      border-radius: 4px;
+      flex-grow: 1;
+      height: 256px;
+      max-width: 408px;
+
+      img {
+        height: 72px;
+        width: 72px;
+      }
+
+      .button {
+        width: 128px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 375px) {
+    h4 {
+      font-size: var(--h5-font-size);
+    }
```

Comment:
please don't do that here
there is  [typography.scss file](https://github.com/openware/yellow-org/blob/2869bca0a9e1c90da207c4c01d7602bc35dd1443/src/css/typography.scss#L222) you can redeclare html global font-size (as soon as font-sizes are set in **rem** the change in global font-size will affect other font-sizes)

you can add more styles for smaller mobile screens